pkg/feature/feature.go

```diff
@@ -59,52 +70,60 @@ func (f *Feature) Apply() error {
 		multiErr = multierror.Append(multiErr, precondition(f))
 	}
 
-	if multiErr.ErrorOrNil() != nil {
-		return multiErr.ErrorOrNil()
+	preconditionsErr := multiErr.ErrorOrNil()
```

Comment:
Hm.... any reason for not preferring assignment within the `if` to limit the lifetime/scope of the variable?

Reply:
Good point, changed in 2607be1.